openlibrary/catalog/works/by_author.py

```diff
@@ -281,9 +281,3 @@ def by_authors():
 
 by_authors()
 sys.exit(0)
```

Comment:
Just returning normally would probably be better.

Reply:
Line 283 should be removed if it is the last line in the file.